testsuite/openshift/kuadrant.py

```diff
@@ -0,0 +1,28 @@
+"""Kuadrant CR object"""
+
+from openshift_client import selector
+
+from testsuite.openshift import OpenShiftObject, modify
+
+
+class KuadrantCR(OpenShiftObject):
+    """Represents Kuadrant CR objects"""
+
+    LIMITADOR = "limitador-limitador"
+
+    @property
+    def limitador(self) -> dict:
+        """Returns spec.limitador from Kuadrant object"""
+        return self.model.spec.get("limitador", {})
+
+    @modify
+    def update_limitador(self, replicas: dict):
```

Comment:
Since we decided to have generic update function with not preprocessing, my previous request to add `replicas` is again outdated, sorry for that. Now it can be again arbitrary dict